lib/service/accountservice.php

```diff
@@ -120,12 +120,8 @@ private function buildUnifiedAccount($userId) {
 	 * @return Account[]
 	 */
 	public function updateSignature($accountId, $currentUserId, $signature) {
-		try {
-			$account = $this->mapper->find($currentUserId, $accountId);
-			$account->setSignature($signature);
-			return $this->mapper->update($account);
-		} catch(Exception $e) {
```

Comment:
Hm. The service should catch implementation specific exceptions (e.g. DB exception) and throw an own exception instead. Maybe something like ``ServiceException``? Example: https://github.com/nextcloud/news/blob/master/lib/Service/ServiceException.php